addon-test-support/setup-intl.ts

```diff
@@ -1,5 +1,16 @@
 import addTranslations from './add-translations';
 import { missingMessage } from './-private/serialize-translation';
+import { Translations } from '../addon/models/translation';
+import IntlService from '../app/services/intl';
+
+interface Hooks {
+  beforeEach: (fn: Function) => any;
+}
+
+interface ApplicationContext {
+  owner: any;
+  intl?: IntlService;
+}
```

Comment:
```ts
import { TestContext } from "ember-test-helpers";

export interface IntlTestContext extends TestContext {
  intl: IntlService;
}
```

This would allow consumers to do this in their tests:

```ts
import { setupIntl, IntlTestContext } from 'ember-intl/test-support';

module('foo', function() {
  setupIntl(this);

  test('foo', (this: IntlTestContext, assert) {
    this.intl.setLocale('en-US');
    // ...
  })
});
```